edx_proctoring/static/proctoring/spec/proctored_exam_add_allowance_spec.js

```diff
@@ -79,15 +79,16 @@ describe('ProctoredExamAddAllowanceView', function() {
      </tr>\n        <tr>\n            <td>\n                <label><%- gettext("Select Exam Type") %></label>\n            </td>\n        </tr>\n        <tr>\n            <td>\n                <select id="exam_type">\n                    <option value="proctored_exam">\n                        <%- gettext("Proctored Exam") %>\n                    </option>\n                    <option value="timed_exam">\n                        <%- gettext("Timed Exam") %>\n                    </option>\n                </select>\n            </td>\n        </tr>\n        <tr>\n            <td>\n                <label><%- gettext("Select Exams") %></label>\n            </td>\n        </tr>\n        <tr>\n            <td>\n                <select multiple id=\'proctored_exam\' class="exam_dropdown">\n                    <option hidden selected value=default> <%- gettext("Choose Exams Below") %> </option>\n                    <% _.each(proctored_exams, function(proctored_exam){ %>\n                    <option value="<%= proctored_exam.id  %>">\n                    <%- interpolate(gettext(\' %(exam_display_name)s \'), { exam_display_name: proctored_exam.exam_name }, true) %>\n                    </option>\n                    <% }); %>\n                </select>\n                <select multiple id=\'timed_exam\' class="exam_dropdown" style="display:none;">\n                    <option hidden selected value=default> <%- gettext("Choose Exams Below") %> </option>\n                    <% _.each(timed_exams, function(timed_exam){ %>\n                    <option value="<%= timed_exam.id  %>">\n                    <%- interpolate(gettext(\' %(exam_display_name)s \'), { exam_display_name: timed_exam.exam_name }, true) %>\n                    </option>\n                    <% }); %>\n                </select>\n            </td>\n        </tr>\n        <tr>\n            <td colspan="3">\n                <div id="selected_exams" ></div>\n            </td>\n        </tr>\n        <tr>\n            <td>\n                <label><%- gettext("Allowance Type") %></label>\n            </td>\n        </tr>\n        <tr>\n            <td>\n                <select id="allowance_type">\n                    <% _.each(allowance_types, function(allowance_type){ %>\n                    <option value="<%= allowance_type[0] %>">\n                        <%= allowance_type[1] %>\n                    </option>\n                    <% }); %>\n                </select>\n            </td>\n        </tr>\n        <tr>\n            <td>\n                <label id=\'allowance_value_label\'><%- gettext("Input Additional Minutes as a Positive Number") %></label>\n            </td>\n        </tr>\n        <tr>\n            <td>\n                <input type="text" id="allowance_value" />\n            </td>\n        </tr>\n        <tr>\n            <td>\n                <input id=\'addNewAllowance\' type=\'submit\' value=\'Create Allowance\' />\n            </td>\n        </tr>\n    </table>\n</form>';
```

Comment:
Your unit test needs to be updated. Instead of changing it directly, you can copy and past what you have, and then rename the new pasted test suite.

Revert your changes on the existing test suite to the old code to test against the old views.